client/my-sites/exporter/exporter.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import AdvancedSettings from 'my-sites/exporter/advanced-settings';
+
+export default React.createClass( {
+	displayName: 'Exporter',
+
+	propTypes: {
+		advancedSettings: PropTypes.shape( {
+			isVisible: PropTypes.bool.isRequired
+		} )
```

Comment:
Should probably make `advancedSettings` isRequired since your not checking its existence in your render method and your using `advancedSettings.isVissible`.

Also consider adding `toggleAdvancedSettings` with `isRequired` since you use it as an onclick callback.